docs/src/user_docs/assembly/cryptographic_operations.md

```diff
@@ -7,7 +7,7 @@ Miden assembly provides a set of instructions for performing common cryptographi
 | Instruction    | Stack_input     | Stack_output   | Notes                                      |
 | -------------- | --------------- | -------------- | ------------------------------------------ |
 | hash <br> - *(20 cycles)*  | [A, ...] | [B, ...] | $\{B\} \leftarrow hash(A)$ <BR> where, $hash()$ computes a 1-to-1 Rescue Prime Optimized hash. |
-| hperm  <br> - *(1 cycle)*      | [C, B, A, ...]  | [F, E, D, ...] | $\{D, E, F\} \leftarrow permute(A, B, C)$ <br> where, $permute()$ computes a Rescue Prime Optimized permutation. |
+| hperm  <br> - *(1 cycle)*      | [C, B, A, ...]  | [C', B', A', ...] | $\{A', B', C'\} \leftarrow permute(A, B, C)$ <br> Performs a Rescue Prime Optimized permutation on the top 12 elements of the operand stack, where the top 8 elements are the rate (words C and B), the deepest 4 elements are the capacity (word A), the digest output is the middle elements 4..8 (word B'). |
```

Comment:
I think it is a bit confusing to assume the output to be `{A', B', C'}` when in fact the product of a permutation shouldn't be related whatsoever to the input. We are stating this is a sort of inverted order, but my point is: there is no actual order. `A` shouldn't relate to `A'`; otherwise, the cryptographic properties of RPO would be broken.

What we can do is highlight that the range we commonly pick as digest output is the second word

Reply:
I'm fine with changing it, what is your suggestion?

Reply:
replaced `C',B',A' <- C,B,A` with `F,E,D <- C,B,A`, removed mention to number of field elements